ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -168,10 +169,21 @@
         .withAuthFilePath(makeRelativePathAbsolute(table.getString("auth-file")))
         .withTimeout(table.getLong("timeout"));
 
+    // optional
+    final TomlTable tomlTable = signingTable.get();
+    final Boolean tlsEnabled = tomlTable.getBoolean("tls-enabled", () -> true);
+    builder.withTlsEnabled(tlsEnabled);
+    if (tlsEnabled) {
```

Comment:
Is this conditional important? Would be much nicer to just chain the withs using the builder.

Reply:
Not that important, just make it technically correct.

Reply:
done